src/overlays/actors/ovl_Dm_Bal/z_dm_bal.h

```diff
@@ -8,11 +8,22 @@ struct DmBal;
 typedef void (*DmBalActionFunc)(struct DmBal*, GlobalContext*);
 
 typedef struct DmBal {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x44];
-    /* 0x0188 */ DmBalActionFunc actionFunc;
-    /* 0x018C */ char unk_18C[0x1B0];
-} DmBal; // size = 0x33C
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ DmBalActionFunc actionFunc;
+    /* 0x18C */ f32 unk18C;
+    /* 0x190 */ f32 unk190;
+    /* 0x194 */ f32 unk194;
+    /* 0x198 */ s32 unk198;
+    /* 0x19C */ Vec3s morphTable;
+    /* 0x1A2 */ char pad1A2[0xC0];
+    /* 0x262 */ Vec3s joinTable;
+    /* 0x268 */ char pad268[0xCC];
```

Comment:
Same comment as in the `Hata` pr, https://github.com/zeldaret/mm/pull/844#discussion_r891677256